apps/admin/cortex-xdr/cortex-xdr.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: helm.toolkit.fluxcd.io/v2
+kind: HelmRelease
+metadata:
+  name: cortex-xdr
+  namespace: admin
+spec:
+  timeout: 20m
+  releaseName: cortex-xdr
+  chart:
+    spec:
+      chart: cortex-xdr
+      version: 1.7.0
+      sourceRef:
+        kind: HelmRepository
+        name: cortex-xdr
+        namespace: admin
+  interval: 1m
+  values:
+    namespace:
+      name: admin
+      create: false
+    daemonset:
+      image:
+        repository: europe-west2-docker.pkg.dev/xdr-uk-2104535898207/agent-docker/cortex-agent
```

Comment:
Is this the only repo available?

Reply:
Helm chart comment is and the value is blank so there is no default:
`# image repository url, can be retrieved from the tenant`

This is the one provided by MIP so I'm not sure if others are available.

Reply:
Fine for POC for now I guess if they gave it - maybe worth asking before we begin the epic if there is a closer to home one or similar